docs/reference/migration/migrate_8_0/snapshots.asciidoc

```diff
@@ -12,7 +12,51 @@
 [float]
 === Get snapshots response format is changed
 It's possible to get snapshots from multiple repositories in one go. The response format has changed
-and now contains separate response for each repository. See <<modules-snapshots>> for more information.
+and now contains separate response for each repository.
+
+For example, requesting one snapshot from particular repository
+
+[source,js]
+-----------------------------------
+GET _snapshot/repo1/snap1
+-----------------------------------
+// CONSOLE
+// TEST[skip:no repo and snapshots are created]
+
+produces the following response
+
+[source,js]
```

Comment:
It doesn't look like this affects the html output or the CI, but adding the delimiter will help with consistency.

```suggestion
[source,js]
-----------------------------------
```